examples/db-templates/README.md

```diff
@@ -39,32 +39,48 @@ local path or an URL. Example:
 
     $ oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json
     --> Deploying template mongodb-ephemeral for "https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json"
-     With parameters:
-      DATABASE_SERVICE_NAME=mongodb
-      MONGODB_USER=userJNX # generated
-      MONGODB_PASSWORD=tnEDilMVrgjp5AI2 # generated
-      MONGODB_DATABASE=sampledb
-      MONGODB_ADMIN_PASSWORD=8bYEs8OlNYhVyMBs # generated
+
+      mongodb-ephemeral
+      ---------
+      MongoDB database service, without persistent storage. WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
+
+      * With parameters:
+         * Memory Limit=512Mi
+         * Namespace=openshift
+         * Database Service Name=mongodb
+         * MongoDB User=userVL5 # generated
+         * MongoDB Password=t6M0YV22sdwehTfn # generated
+         * MongoDB Database Name=sampledb
+         * MongoDB Admin Password=Q3CehePlq3MGleUx # generated
+
     --> Creating resources ...
-    Service "mongodb" created
-    DeploymentConfig "mongodb" created
+        service "mongodb" created
+        deploymentconfig "mongodb" created
     --> Success
-    Run 'oc status' to view your app.
+        Run 'oc status' to view your app. 
```

Comment:
Perhaps we should just remove the example output, instead of trying to update it. It will keep getting outdated as we introduce changes to the output format of `oc new-app`.

I'd propose nothing more than:

```
$ oc new-app https://raw.githubusercontent.com/openshift/origin/master/examples/db-templates/mongodb-ephemeral-template.json
```

@bparees WDYT?